build.sbt

```diff
@@ -1,4 +1,4 @@
-ThisBuild / kantanProject := "codecs"
+ThisBuild / kantanProject := "kantan.codecs"
```

Comment:
Published artifacts (on Github packages) were misnamed. Did I miss something?

Reply:
That's a little odd - what names were you getting? this should be unnecessary.

Reply:
If I remember correctly, `kantan` was not in the artifact name. I'll remove this change and if you have issues when publishing, then we'll fix it.

Reply:
Reverted!